tests/openebs-control-plane-validation_test.go

```diff
@@ -0,0 +1,124 @@
+	restclient "k8s.io/client-go/rest"
+)
+
+var (
+	kubeconfig string
+	config     *restclient.Config
+	client     *kubernetes.Clientset
+	clientSet  *chaosClient.LitmuschaosV1alpha1Client
+	err        error
+)
+
+func TestChaos(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = BeforeSuite(func() {
+	var err error
+	kubeconfig = os.Getenv("HOME") + "/.kube/config"
+	config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get config")
+	}
+
+	client, err = kubernetes.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get client")
+	}
+
+	clientSet, err = chaosClient.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get clientSet")
+	}
+
+	err = v1alpha1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+})
+
+//BDD Tests for openebs pool container failure
+var _ = Describe("BDD of openebs pool container failure experiment", func() {
+
+	// BDD TEST CASE 1
+	Context("Check for the components of openebs", func() {
+
+		It("Should check for creation of runner pod", func() {
+
+			
+			//Creating rbac for the experiment
+			By("Creating rbac for the experiment")
+			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-control-plane-validation/rbac.yaml", "-n", "openebs").Run()
+			Expect(err).To(BeNil(), "fail to create chaos experiment")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			//Creating Chaos-Experiment
+			By("Creating Experiment")
+			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-control-plane-validation/rbac.yaml", "-n", "openebs").Run()
+			Expect(err).To(BeNil(), "fail to create chaos experiment")
+			if err != nil {
+				fmt.Println(err)
+			}
+			
+			//Creating Chaos Engine
+			By("Creating Chaos Engine")
+			err = exec.Command("kubectl", "apply", "-f", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/openebs/openebs-control-plane-validation/engine.yaml", "-n", "openebs").Run()
+			Expect(err).To(BeNil(), "fail to create chaos engine")
+			if err != nil {
+				fmt.Println(err)
+			}
+
+			time.Sleep(5 * time.Second)
+
+			fmt.Println("Chaosengine created successfully...")
+
+			//Fetching engine-nginx-runner pod
+			runner, err := client.CoreV1().Pods("openebs").Get("control-plane-chaos-runner", metav1.GetOptions{})
+			fmt.Printf("name : %v \n", runner.Name)
+			for i := 0; i < 30; i++ {
+				if string(runner.Status.Phase) != "Succeeded" {
+					time.Sleep(10 * time.Second)
+					runner, _ = client.CoreV1().Pods("openebs").Get("control-plane-chaos-runner", metav1.GetOptions{})
+					fmt.Printf("Current Runner is in %v State, Please Wait ...\n", runner.Status.Phase)
```

Comment:
`fmt.Printf("Currently the  Runner pod is in %v State, Please Wait ...\n", runner.Status.Phase)`

Reply:
Done